spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1330,6 +1330,67 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
               None
             }
           }
+        case StructsToString(options, child, timezoneId) =>
+  if (options.nonEmpty) {
```

Comment:
It looks like the formatting is off here. You can run `make format` to fix this,